src/argilla_sdk/records/_resource.py

```diff
@@ -289,6 +190,40 @@ def to_dict(self) -> Dict[str, Dict]:
             "_server_id": str(self._model.id),
         }
 
+    @classmethod
+    def from_dict(cls, data: Dict[str, Dict], dataset: Optional["Dataset"] = None) -> "Record":
+        """Converts a dictionary to a Record object.
+        Args:
+            data: A dictionary representing the record.
+            dataset: The dataset object to which the record belongs.
+        Returns:
+            A Record object.
+        """
+        fields = data.get("fields", {})
+        metadata = data.get("metadata", {})
+        suggestions = data.get("suggestions", {})
+        responses = data.get("responses", {})
+        vectors = data.get("vectors", {})
+        record_id = data.get("id", None)
+
+        suggestions = [Suggestion(question_name=question_name, **value) for question_name, value in suggestions.items()]
+        responses = [
+            Response(question_name=question_name, **value)
+            for question_name, _responses in responses.items()
+            for value in _responses
+        ]
+        vectors = [Vector(name=vector_name, values=values) for vector_name, values in vectors.items()]
```

Comment:
Why do we use `values` here?

Reply:
in the sense of it being plural,

Reply:
We follow the server's use of `vector_values` and use `values`. I assume because the attribute contains an series.